packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,12 +101,25 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether the announcement should interrupt any existing announcement,
+  /// or queue after it.
+  ///
+  /// On the web this option uses the aria-live level to set the assertiveness
+  /// of the announcement. On iOS, Android, Windows, Linux, macOS, and Fuchsia
+  /// this option currently has no effect.
+  final Assertiveness assertiveness;
+
   @override
   Map<String, dynamic> getDataMap() {
-    return <String, dynamic>{
+    final Map<String, dynamic> dataMap = <String, dynamic> {
       'message': message,
       'textDirection': textDirection.index,
     };
+
+    if (assertiveness == Assertiveness.assertive) {
```

Comment:
Please use switch/case for enums: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values